.github/workflows/build-and-test.yml

```diff
@@ -117,20 +144,35 @@ jobs:
            -DCMAKE_C_COMPILER_LAUNCHER=ccache                                                          \
            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache                                                        \
            ${GITHUB_WORKSPACE}
-      - run: ccache -p
-      - run: ccache -z
-      - name: CMake Build
-        working-directory: ./build
+      - name: CMake Build (Ubuntu)
+        if: ${{ startsWith(matrix.config.os, 'ubuntu') }}
+        working-directory: ./build_${{ matrix.config.conan_profile }}
         run: cmake --build .
-      - name: CCache Stats
-        run: ccache -s
-      - name: CMake Test
-        working-directory: ./build
+      - name: CMake Test (Ubuntu)
+        if: ${{ startsWith(matrix.config.os, 'ubuntu') }}
+        working-directory: ./build_${{ matrix.config.conan_profile }}
         run: CTEST_OUTPUT_FAILURE=1 ARGS="-E IntegrationTest" cmake --build . --target test
+      - name: Install conan config (Windows)
+        if: ${{ startsWith(matrix.config.os, 'windows') }}
+        run: |
+          ./third_party/conan/configs/install.ps1
+          py -3 -m conans.conan config install .\third_party\conan\configs\windows_ci\
+      - name: Build & Test (Windows)
+        if: ${{ startsWith(matrix.config.os, 'windows') }}
+        env:
+          Qt5_DIR: "C:\\Qt\\5.15.2\\msvc2019_64"
+          CMAKE_CXX_COMPILER_LAUNCHER: "ccache"
+          CMAKE_C_COMPILER_LAUNCHER: "ccache"
+        run: ./build.ps1 ${{ matrix.config.conan_profile }}
```

Comment:
I would consider calling Conan directly here. That simplifies the build - especially because you could have a static build directory path (build/ for example - as before). These are the commands:

```
py -3 -m conans.conan install -pr:b msvc2019_release -pr:h ${{ matrix.config.conan_profile }} -if build/ --build outdated --update ./
py -3 -m conans.conan build -bf build/
```

Reply:
Done (also used ${{ matrix.config.conan_profile }} for the first msvc2019_release).